higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -175,6 +175,11 @@ <h2 class="subheading">Confirmation</h2>
       {% endfor %}
     };
 
+    let other_varsity = '999999';
```

Comment:
I don't really like this idea of adding a row to the DB with a magic number ID.

Can we not rather just add an "Other" option to the list in the web form, with a null ID, and not add anything to the database?

So you have
```
1: Boland
2: College of Cape Town
null: Other
```
As the options of the dropdown list

Reply:
@rudigiesler thanks for this, couldn't wrap my head around it